src/elements/content-sharing/types.js

```diff
@@ -174,3 +174,12 @@ export interface FetchItemProps extends BaseFetchProps {
 export interface FetchCollaboratorsProps extends BaseFetchProps {
     collaborators: Collaboration[];
 }
+
+export interface SharedLinkSettings {
```

Comment:
maybe use a partial of `LinkSettingsFormData` from usm instead of redefining this?